docs/docs/concepts/repos.md

```diff
@@ -30,20 +29,19 @@ If the directory is not a cloned Git repo, it uploads the entire directory.
 
 Uploads are limited to 2MB. Use `.gitignore` to exclude unnecessary files from being uploaded.
 
-### Initialize as a regular directory
+### Initialize as a local directory
 
-If the directory is a cloned Git repo but you want to initialize it as a regular directory, use
-`--local` with [`dstack init`](../reference/cli/dstack/init.md).
+If the directory is a cloned Git repo but you want to initialize it as a regular local directory,
+use `--local` with [`dstack init`](../reference/cli/dstack/init.md).
 
-## Override the repo
+## Specify the repo
```

Comment:
IMO the semantics here is a bit unclear

Reply:
I suggest "Inline repo initialization" instead of "Specify the repo"

Reply:
The section is less about repo init and more about the possibility to specify repo for `dstack apply`. I'd not focus on initialization since it's not the user goal. Maybe "Use repos with dstack apply"?